src/constants/markets.ts

```diff
@@ -79,9 +84,16 @@ export const MARKET_FILTER_OPTIONS: Record<
   [MarketFilters.ALL]: {
     label: STRING_KEYS.ALL,
   },
+  [MarketFilters.FAVORITED]: {
+    label: undefined, // replace with star icon
+  },
```

Comment:
maybe we can update `label` to be React.ReactNode and add `labelStringKey` as well. That way we don't need to pass `undefined` and use custom handling in the `<MarketFilter>`